straxen/plugins/event_processing.py

```diff
@@ -36,6 +35,11 @@
                  default=True, type=bool,
                  help='If true exclude S1s as triggering peaks.',
                  ),
+    strax.Option(
+        name='event_s1_min_coincidence',
+        default=2, infer_type=False,
+        help="Event level S1 min coincidence. Should be >= s1_min_coincidence "
```

Comment:
I am wondering if we should not add the condition as an assert statement.

Reply:
sure can add - not strictly the scope of this PR though